src/clients/secretsManagerService.test.js

```
@@ -0,0 +1,151 @@
+import { beforeEach, describe, expect, jest, test } from '@jest/globals';
```

ESLint

[error] 'jest' is defined but never used.

(no-unused-vars)